immutablecollections/newsfragments/42.breaking

```diff
@@ -0,0 +1 @@
+ImmutableList has been removed in favor of `tuple`.
```

Comment:
Can you elaborate a little on the reasoning for this?(that now that we aren't attempting runtime type-checking,it has no advantage)

Reply:
Maybe it should say "ImmutableList no longer exists"? That was the intended message.

Reply:
I think what you have is fine, just amend it like this:

```suggestion
ImmutableList has been removed in favor of `tuple`.
`ImmutableList` has been removed in favor of `tuple`. Now that we are no longer targeting support for runtime type-checking of collections, `ImmutableList` had no advantage of `tuple` for any purpose.
```